keyboards/dz60/keymaps/_bonfire/dz60.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
```

Comment:
The contents of this file could actually be moved into `layers.c`, as that's the only place it's needed.